content/posts/developing-versionless-graphql-apis.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Developing versionless APIs with GraphQL"
+date: "2020-07-22"
+author: "Bing Xiao"
+github: xo426017
```

Comment:
The author/github section here needs to be of the form:

```text
authors:
- github: jpreese
  name: "John Reese"
```

Note that it starts with `authors:` and then the github/name fields below that.

Reply:
You can see a current example here for reference:
https://raw.githubusercontent.com/plexsystems/plexsystems.github.io/develop/content/posts/container-structure-test.md

Reply:
Fixed